data/api.py

```diff
@@ -253,6 +254,13 @@ class JobOutputDirViewSet(viewsets.ModelViewSet):
     queryset = JobOutputDir.objects.all()
     serializer_class = JobOutputDirSerializer
 
+    @detail_route(methods=['post'], url_path='readme-url')
```

Comment:
Doesn't the detail route need to specify the serializer is a `DDSFileURLSerializer` instead of a `JobOutputDirSerializer`?